src/bots/ama/index.js

```diff
@@ -0,0 +1,40 @@
+import client from '../../client';
+import { parseCommandString } from "../../utils/index";
+import { escapedBackticks } from "../../utils/embed";
+import addEvent from './subCommands/addEvent';
+import listEvents from './subCommands/listEvents';
+
+const subcommands = {'add': addEvent, 'list': listEvents}
+
+client.on('ama', () => {
```

Comment:
can we move basically all of this to a `handler.js` similar to comp bot and pins bot? i kinda of like the pattern of keeping index.js very clean + also not having everything in the event listener callback

Reply:
isn't that just adding redundant abstraction though? All index.js -> handler.js by the looks of it

Reply:
Yeah I can't say you're wrong, it was just a style I liked. We can do it this way too

Reply:
I don't mind doing either, just would prefer some idea to what else index.js can be used for. Like if it will be used for anything else then yeah use handler for the sake of single use principle

Reply:
Yeah we can keep it like this for now